common/src/main/java/com/wynntils/models/containers/type/wynncontainers/BlockBankContainer.java

```diff
@@ -16,7 +18,7 @@ public BlockBankContainer() {
     }
 
     @Override
-    public boolean supportsAdvancedSearch() {
-        return false;
+    public List<ItemProviderType> supportedProviderTypes() {
+        return List.of(ItemProviderType.COUNTED);
```

Comment:
We don't parse any items in the block bank to WynnItem's so unless you're planning to do that in the future I don't think this is needed